mewarpx/mewarpx/mwxrun.py

```diff
@@ -151,6 +154,10 @@ def get_dt(self):
         """Return the timestep."""
         return _libwarpx.libwarpx.warpx_getdt(self.lev)
 
+    def get_t(self):
+        """Return the simulation time."""
+        return (self.get_it() - 1.0) * self.get_dt()
```

Comment:
I assume you've checked that WarpX step 1 has time = 0? I'm too lazy to right now...

Reply:
Haha I did.